qemu/tests/cfg/ctrl_vlan.cfg

```diff
@@ -0,0 +1,19 @@
+- ctrl_vlan: install setup image_copy unattended_install.cdrom
```

Comment:
You can delete content after colon. It doesn't work for avacodo.

Reply:
Thanks for review, will remove it.